src/Chapter1.hs

```diff
@@ -635,8 +646,10 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+  | div n 10 == 0 = n
```

Comment:
You are almost there! 💪🏼  This is a nice solution, however, it won't work on the negative input.

Reply:
I did putting the abs on, but I feel it can be reduced